modules/util/portal-tools-db-upgrade-client/src/main/java/com/liferay/portal/tools/db/upgrade/client/AppServer.java

```diff
@@ -51,7 +51,7 @@ public static AppServer getTCServerAppServer() {
 
 	public static AppServer getTomcatAppServer() {
 		return new AppServer(
-			"../../tomcat-8.0.32", "/bin", "/lib", "/webapps/ROOT", "tomcat");
+			"../../tomcat-9.0.6", "/bin", "/lib", "/webapps/ROOT", "tomcat");
```

Comment:
@david-truong, what do you suggest to do for this? Should we have a single version of the tool, fork it, make it configurable, etc.?

Reply:
It's sort of configurable already in that when you run the client you can change the value.  That's just used for the default values

Reply:
Yes it is okay.  Will it get backported to 7.0 also?  That might make it a little annoying for 7.0 users

Reply:
only to 7.1 and master then!

Reply:
Sorry, missed Dave's response as I was typing. Thanks guys.